### PR TITLE
Add observerNotes field

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql [hunk 1]
```diff
@@ -3104,6 +3104,10 @@ input ObservationPropertiesInput {
   """
   groupIndex: NonNegShort
 
+  """
+  Set the notes for  thhe observer
+  """
+  observerNotes: String
```
Comment (cquiroz) [inline modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql:3110]: I'm not sure if this should be `NonEmptyString`. Any opinions?

File: modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala [hunk 2]
```diff
@@ -901,6 +909,11 @@ object ObservationService {
           },
           SET.status.map(upStatus),
           SET.activeStatus.map(upActive),
+          SET.observerNotes match {
+            case Nullable.Null  => Some(upObserverNotes(None))
+            case Absent         => None
+            case NonNull(value) => Some(upObserverNotes(Some(value)))
+          },
```
Comment (swalker2m) [inline modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala:916]: A convenient way to do this is with `foldPresent` on `Nullable`:

```
SET.observerNotes.foldPresent(upObserverNotes)
```

File: modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql (hunk 1 above)
Comment (swalker2m) [reply modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql:3110]: I don't think it matters that much, but we made the subtitle `NonEmptyString` so I guess we could be more consistent with that.

**Review by swalker2m — APPROVED** [review]
This looks great.  :+1:

**Pull request merged**